### ISSUE TITLE
Plan to shoot more videos from home

### ISSUE THREAD

**Issue by rrrutledge** [InnerSourceCommons/InnerSourceLearningPath#274]
We'd like to continue to shoot more training videos around InnerSource.  Since we're in time of lockdown, we'd need these recordings to be done in the home.

@claredillon brought up the idea home video production kits that can be mailed to presenters for self-filming in their own home.  We should investigate this idea further and come up with a realistic plan of how this would work.

**Comment by claredillon** [comment]
Status Update: Have compiled a list of equipment needed for 650 EUR. Issue at the moment is high def camera availability (shortage of 1080 webcams). I am going to purchase to test set-up over next few weeks (dependent on availability/delivery times). Will share results. Propose if results are satisfactory, we can put together 'recording kit in a box'.

**Comment by rrrutledge** [comment]
Sounds great.  How much are shipping costs?  Do we need a set for North America and a set for Europe because of trans-Atlantic shipping costs?


Russ

**Comment by rrrutledge** [comment]
Also, Clare, can you attach or link to the specific items that would be in the kit (and their cost)?

**Comment by claredillon** [comment]
Here is my current working list (work in progress). I will edit as I check availability and get feedback on alternatives. If anyone has alternative suggestions or additions - be great to include.

https://docs.google.com/spreadsheets/d/12picMGF3mPBXztTGHtvGzA9dbLAA2Pv1XbZso21ckj0/edit?usp=sharing

**Comment by claredillon** [comment]
> Sounds great. How much are shipping costs? Do we need a set for North America and a set for Europe because of trans-Atlantic shipping costs?



> Sounds great. How much are shipping costs? Do we need a set for North America and a set for Europe because of trans-Atlantic shipping costs?

I would say yes to having set in each region. We can estimate shipping costs once I get everything together (and weigh them! ;). The bulkiest items are the lights/backdrop. Also will have to look at getting a case or something to protect them in shipping. But first step will be to make sure the set-up does the job.

**Comment by claredillon** [comment]
No news here - will update as soon as kit arrives and has been tried out.

**Comment by rrrutledge** [comment]
Next step is to catalogue equipment and start creating a tutorial on how to shoot videos.

**Comment by rrrutledge** [comment]
Hi, @claredillon, in our last meeting there was [interest from a community member](https://github.com/InnerSourceCommons/InnerSourceLearningPath/issues/294) in starting on another training segment.  Nothing's happening immediately tomorrow, but we wanted to ask again about your progress in being able to provide a home filming kit for community members to use to create more training content.

**Comment by lenucksi** [comment]
There used to be a good, public instructional video on video-shooting by O'Reilly. Does anyone remember where it used to be?

**Comment by rrrutledge** [comment]
Here is [the video](https://app.oreilly.com/authors/welcomekit/video.csp), @lenucksi

**Comment by Danese** [comment]
In re equipment...I have been gifted three separate USB cameras in the past few months (because it's often a gift for keynote speakers). The quality of the image varies WIDELY between cameras sold by the same brand. My favorite is my LogiTech BRIO HD, which lists in Europe for ~€280 but is often backordered at that price. You can easily find one for ~€480 atm... I also own two other Logitech cameras, and neither of them produce a better picture than the ambient camera in my new MacBook Pro 13 (so it's not worth using them).

For microphone, most folks I know have a Blue Yeti. They are durable and easy to use. They do not have a speaker, so you must use computer audio out or headphones of some type..

For lighting, GitHub sent me a fancy LED (so programmable for color) scatter light that was designed to be placed just behind a monitor to cast light on the face. I shoot it at the ceiling and get reflected light (much kinder).

**Comment by rrrutledge** [comment]
Great!  I am thinking that @claredillon will take in all of this and then produce a kit with instructions that can be mailed to commons participants so that they can shoot more videos for the `#learning-path` from their home.

**Comment by lenucksi** [comment]
The insight on the cameras is great, thanks @Danese. I'm using an Auna CM 900 microphone now that I've heard sounds reasonable. It's USB and costs about 40€ in Germany.

**Comment by claredillon** [comment]
A reminder that I started a list of equipment and prices here: https://docs.google.com/spreadsheets/d/12picMGF3mPBXztTGHtvGzA9dbLAA2Pv1XbZso21ckj0/edit?usp=sharing 
If folks have alternative suggestions - be great to also add them. I'll put in the ones mentioned above.

**Comment by claredillon** [comment]
No update again this week - sorry - still on the to do.

**Comment by claredillon** [comment]
Latest update: My latest update on "filming to O'Reilly quality" is that I haven't managed to recreate the dual camera set up (capturing standard and close up feeds simultaneously to allow editors to mix for effects later). From discussions with some others who are doing this - you'd need a digital SLR camera that can be set up with the correct focus. In addition, my choice of webcam would not be recommended - as I am experiencing issues with it going out of focus. Danese seems to have a more reliable version. So in summary, my set-up could be used - but it does not reach the quality you achieved with O'Reilly. We can look at the more sophisticated alternatives when the foundation can access the funds available to purchase equipment. I'll get more details about the higher spec'd equipment from those I know who are using it.

**Comment by tsadler1988** [comment]
@rrrutledge hoping to look at this in December (suggestion: "How to incorporate InnerSource into Sprint planning").

**Comment by claredillon** [comment]
Latest update: following on from update above - confirming that someone else will need to investigate camera options (and possibly appropriate audio options). Because the we now have funds coming into ISC bank account - there is an option to purchase/experiment with different recording set-ups. 

Open questions/points: 

- Do we need the exact same set up as the O'Reilly recordings (specifically two camera angles)?
- It was suggested in a chat with Daniel we should also start creating a set of resources describing optimal recording behaviours. Should this be part of this card? Or separate?

**Comment by rrrutledge** [comment]
> * Do we need the exact same set up

Just needs to be an "acceptable" level of quality (whatever we determine that to be).  Two-camera not strictly necessary.

**Comment by lenucksi** [comment]
Looks like the fancy camera will nuke any foreseeable budget

**Comment by Danese** [comment]
I’m still finding the Logitech BRIO HD to be a fine USB camera (for Mac at
least, but I’m bringing one back for Clare so we’ll soon learn about PCs).
$250 in the US

**Comment by lenucksi** [comment]
Here's another feat that I found out a different way:
- Many of the small still camera manufacturers (What would be known in German as "Knipse") now have provided clean HDMI output or even streaming video over USB. 
- Those cameras are around 300€ and have excellent glass optics, optical zoom, good autofocus and can also do still images as they were designed for pretty much that. 
- If we're using a HDMI transmission this would require an additional HDMI->USB adapter which is reasonably priced in the USB case there only has to be driver support.

From what I've read a bunch of Sony cameras are really popular here, the usual Canon suspects provide some options here too.

Given that we might end up with about 300-400€ total for real optics this might really be worth a look, I'll likely do a test run in a near future.
